src/components/custom/ZwaveGraph.vue

```diff
@@ -769,10 +771,11 @@ export default {
           failed: node.failed,
           title:
             '<b>' +
-            nodeName +
+            node._name +
```

Comment:
@ahochsteger Shouldn't this be `nodeName` instead? BTW I was planning to remove the tooltip and use the popup I was using in the old mesh version. The code is still there, we should detect the node click (it's already detected but nothing happens atm) and when clicked use the code in Mesh.vue to display the draggable info panel. Thoughts?

Reply:
@robertsLando you're right - this was a left-over from a previous variant that used `_name` since it already contained the location. It's done.
Yes, it would be good to get the view back that was shown in the old graph. This would allow us to put more functionality there that is helpful in the graph view.

Reply:
Would you add that in this pr?

Reply:
@robertsLando  I can try.

While we are at it, I've got a question concerning which functionality of `ManagedItems` we'd like to sync between the nodes table and the mesh graph.

What I'd like to see synced is:
* Node filter
* Node grouping

I'm unsure about:
* Node selection
* Visible node properties (or table columns)

Also I'm unsure, if we always want to sync this functionality or if the user should be able to decide if they are linked together.
In case we always sync, I'm thinking about making the managed nodes stuff available using Vuex to access it from both the nodes table and the mesh graph.
What do you think?

Reply:
I wouldn't sync anything bewtween the two tables, I would simply use the same underlying logic for filtering